app/views/devise/two_step_verification/new.html.erb

```diff
@@ -0,0 +1,88 @@
+<h1>Set up 2-step verification</h1>
```

Comment:
This page has a title of `| GOV.UK Signon`. Would you mind calling `content_for(:title)`, probably with the same copy?

The page for entering the code has the same problem, but we need to override that template to solve that problem, so I'm happy to wait for the frontend ticket for that /cc @fofr .